nex/nex.go

```diff
@@ -14,8 +16,30 @@ var (
 	COMMIT    = ""
 	BUILDDATE = ""
 
-	ncli  = fisk.New("nex", "")
+	blue = color.New(color.FgBlue).SprintFunc()
+
+	ncli = fisk.New("nex", fmt.Sprintf("%s\nNATS Execution Engine CLI Version %s\n", blue(Banner), VERSION))
+	_    = ncli.Author("Synadia Communications")
```

Comment:
I really hate that the linter makes us do this, because `ncli.Author(...)` looks so much cleaner.

Reply:
There _IS_ a "dumb linter" commit in this PR